client/landing/gutenboarding/components/domain-picker-button/index.tsx

```diff
@@ -44,89 +46,86 @@ const DomainPickerButton: React.FunctionComponent< Props > = ( {
 } ) => {
 	const buttonRef = React.createRef< HTMLButtonElement >();
 
-	const domainSuggestions = useDomainSuggestions( { locale: useI18n().i18nLocale, quantity: 10 } );
-
-	const domainCategories = useSelect( ( select ) =>
-		select( DOMAIN_SUGGESTIONS_STORE ).getCategories()
-	);
+	const [ domainSearch, setDomainSearch ] = React.useState< string >( '' );
 
-	const { domainSearch, domainCategory } = useSelect( ( select ) =>
```

Comment:
When moving `domainCategory` state into `DomainPicker` component, the selected domain category is no longer persisted after closing Domain Modal.

Reply:
I'm not sure what you mean?

Reply:
Currently in production: after I've selected a domain category and then confirm de selection, next time I open the Domain Modal, I find that category already selected.

On this branch: category selection isn't saved from one DomainModal interaction opening to another: https://cloudup.com/cgXioTPqRx6